shell/platform/darwin/macos/framework/Source/AccessibilityBridgeMacDelegate.mm

```diff
@@ -265,9 +266,10 @@
     case ui::AXEventGenerator::Event::CHILDREN_CHANGED: {
       // NSAccessibilityCreatedNotification seems to be the only way to let
       // Voiceover pick up layout changes.
+      FlutterAppDelegate* appDelegate = (FlutterAppDelegate*)[NSApp delegate];
       events.push_back({
           .name = NSAccessibilityCreatedNotification,
-          .target = [NSApp mainWindow],
+          .target = appDelegate.mainFlutterWindow,
```

Comment:
I am not sure sometimes the `[NSApp mainWindow]` will be null, and they will be different. but the appDelegate.mainFlutterWindow seems to be the most preferable way of grabbing the nswindow.

Reply:
You appear to have the engine as an ivar; given that you can get the associated view then get its window.

Reply:
It only works when I send the NSAccessibilityCreatedNotification to the NSWindow of the view. The API is a black box and I have no idea why.

Reply:
done